dcpy/connectors/edm/publishing.py

```diff
@@ -194,18 +194,19 @@ def publish(
     acl: s3.ACL,
     publishing_version: str | None = None,
     keep_draft: bool = True,
-    target_bucket: str | None = None,
     max_files: int = s3.MAX_FILE_COUNT,
+    latest: bool = False,
```

Comment:
just curious why the default here is false but it's true in the github action. maybe so any potential dcpy tests wouldn't overwrite the more human/manual concept of "latest"?

Reply:
Something like that. Seems good by default (if we're manually doing anything with command line, etc) to not push to multiple places. But if we're doing something more official (using gha) we'll generally be wanting to push to latest.